server/settings/comprefacesettings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Compreface Settings": [
+        {
+            "Domain": "http://127.0.0.1"
+        },
+        {
+            "Port": "8000"
+        },
+        {
+            "API Key": "c31ce2a9-d276-417d-b7e5-bc0f3f496783"
```

Comment:
This is ok to be public?

Reply:
Yeah it's okay to be public. The Compreface server isn't accessible from the outside, and doesn't have to be. The IP address is 127.0.0.1, which is just localhost. I should make it clear in the docs that everything runs local for this, and to not expose this port.